docs/guides/how_to_guides/creating_batches/how_to_configure_a_runtime_data_connector.rst

```diff
@@ -0,0 +1,80 @@
+.. _how_to_guides__creating_batches__how_to_configure_a_runtime_data_connector:
+
+How to configure a RuntimeDataConnector
+==========================================
+
+This guide demonstrates how to configure a :ref:`RuntimeDataConnector<runtime_data_connector_and_runtime_batch_request>`.
+
+.. admonition:: Prerequisites: This how-to guide assumes you have already:
+
+    - :ref:`Set up a working deployment of Great Expectations <tutorials__getting_started>`
+    - :ref:`Understand the basics of Datasources in 0.13 or later <reference__core_concepts__datasources>`
+    - Learned how to configure a :ref:`DataContext using test_yaml_config <how_to_guides_how_to_configure_datacontext_components_using_test_yaml_config>`
```

Comment:
I totally nitpicked this and changed to Data Context (two words) everywhere in the datasource docs yesterday.